### DISCUSSION TITLE
Upload proof of academic status: endless wait

### DISCUSSION THREAD

**Discussion by EvertDB-ORC** [community/community#88553] [GitHub Education] [answered]
### Hi everyone,

Hi all

### Problem description
I want to upload a picture in order to provide proof of my academic status. Unfortunately, I'm unable to finish uploading it: I choose the picture of my teacher's card and press "Process my application"-button so the text of that button changes into "Processing information..." ... and that's all that seems to happen.

My image is a .JPG of 1.24MB and 3084px x 1960px, so it passes the requirements. Browser version is  Edge v120.0.2210.133 (official build) (64-bit).

### What did I try already?

- Retried the same process (a couple of times)
- Completed my profile as one of the help pages suggested
- Retried a couple of times
- Found another help page that instructed to sign-out and sign-in again
- Retried the process
- Cleared my cookies for education.github.com as found on another help page and rotated my image to landscape
- Retried the process

I'm out of options here... 

Thanks in advance for any help! 
Evert

**Comment by vonHartz** [comment]
Have the same issues. Also tried different browsers (Firefox, Chromium) and devices (linux and iOS).

**Comment by niwazukihon** [comment]
I was about to write a post on the same issue. It is stuck on `Processing information` forever.

**Comment by EvertDB-ORC** [comment, marked as answer]
Hi @vonHartz and @arthniwa 

I saw [another issue where a solution was mentioned](https://github.com/orgs/community/discussions/86731#discussioncomment-8157667): make the image really light (I resized to 1080px x 686px, resulting in 133kB).
This worked for me, I hope for you too!

**Comment by EvertDB-ORC** [comment]
Answer found in [this issue]( #86731)... credits to @buczbuska.
